### ISSUE TITLE
can not see old membership level

### ISSUE THREAD

**Issue by aakankshk** [strangerstudios/pmpro-multiple-memberships-per-user#6]
Hi

i have already two membership and i want to multiple membership that user can buy. i just add the add once and see membership level page strange i can not see my old level and even can not add new membership level in group 

is something wrong i did?

Thanks 

![image](https://cloud.githubusercontent.com/assets/25951825/23206699/759f017e-f8a3-11e6-8f51-36637abb5ae6.png)

**Comment by AAnkudovich** [comment]
Same issue here 
your old memberships are still available just go to yourdomaim.com/wp-admin/admin.php?page=pmpro-membershiplevels&edit=`LEVELID`

LevelID needs to be changed to your membership id (they auto increment so if you dont know your ones try 1 then 2 and so on)

**Comment by aakankshk** [comment]
Thanks for such a quick replay and i can see my old membership 

But how can i add that membership in as group as you can see in screen short and even i can not add new membership in demo group

any idea?

**Comment by AAnkudovich** [comment]
Groups are used to determine what memberships can go in parallel and which cant. im afraid i ended up skipping it and restricting users through front end from getting memberships of same group allocation.

**Comment by aakankshk** [comment]
ok
i have scenario like i want give access of two different membership to my users under one access both membership level have different price 

how can i do i do not want option like separate domains

**Comment by AAnkudovich** [comment]
that should work perfectly fine 

you can create a custom pricing table by creating a template for `/membership-account/membership-levels/`

on the bottom point it to checkout pages 
     
    <a href="/membership-account/membership-checkout/?level=15" class="btn btn-primary" style="width: 110px;"> Buy It </a>

You can also disable the button by checking for the membership like so

       <a href="/membership-account/membership-checkout/?level=15" class="btn btn-primary" style="width: 110px;" <?php if (is_user_logged_in()) { if(pmpro_hasMembershipLevel(array(15))){ echo 'disabled';  }} ?>> Buy It </a>

**Comment by aakankshk** [comment]
yes but how it work suppose

page A is only access by Membership A
page B is only access by Membership B

Now if any user purchase the Membership A and he want same access of Page B to if user purchase Membership B than they remove form accessing page A that is what main issue

Thanks

**Comment by AAnkudovich** [comment]
The function i use is deep inside of plugin code and works wonders

    <?php if(pmpro_hasMembershipLevel(array(15))): ?> 
              <div class="page-content">
                <?php the_content(); ?>
                
              </div>
    <?php else: ?> 
               
              <h3>Buy our membership</h3>
              
              
              <?php endif; ?>


`pmpro_hasMembershipLevel(array(15))` will check if user has the membership id 15 (change to your membership number) you can add comma and additional memberships too (ie. `pmpro_hasMembershipLevel(array(15,16,1,7))`)

**Comment by AAnkudovich** [comment]
If membership matches that in if statement the user sees post or page content else it shows `Buy our membership`

**Comment by aakankshk** [comment]
thanks

my only concern is if user have one membership already and purchase the another membership with exist login access can they access both restricted pages

Hope you understand

**Comment by aakankshk** [comment]
thanks 

let me test i am afraid because in back end it show as membership which is last buy that is why i am confuse

**Comment by AAnkudovich** [comment]
U mean in Database it shows singe memberships ?

Check what you see on this /wp-admin/admin.php?page=pmpro-memberslist when looking at an account with 2 memberships it should show something like this (comma sepparated membership names)
![screen shot 2017-02-22 at 10 57 50](https://cloud.githubusercontent.com/assets/10615682/23208663/c907202c-f8ed-11e6-8670-f91e1b09c8c2.png)

**Comment by aakankshk** [comment]
Thanks

but i can see last buy membership i can access the page with associate with membership b (Note: i have not used multiple membership add on)

![image](https://cloud.githubusercontent.com/assets/25951825/23209057/6a0190da-f8ac-11e6-83ae-2184410c052e.png)

**Comment by AAnkudovich** [comment]
yes now if you activate multiple memberships and you buy membership A it should show both there

Note : this addon doesnt work with paypal

**Comment by AAnkudovich** [comment]
alternatively go into your database and run equivalent of this (i use mysql)

INSERT INTO wp_pmpro_memberships_users (user_id,membership_id,initial_payment,cycle_period,status,startdate,enddate) VALUES (30,1,PRICEOFMEMBERSHIP,"","active",'2016-12-01','2017-12-20');

**Comment by aakankshk** [comment]
Thanks

now i active the addons and check with stripe my membership list page blank and memberlist doesn't update

![image](https://cloud.githubusercontent.com/assets/25951825/23209626/dcea3c94-f8ae-11e6-935b-6cfa052176dd.png)


![image](https://cloud.githubusercontent.com/assets/25951825/23209452/1d23f1ac-f8ae-11e6-8a2a-2dea84dc7d32.png)

**Comment by aakankshk** [comment]
front end

![image](https://cloud.githubusercontent.com/assets/25951825/23209726/410ce456-f8af-11e6-9787-8ddfe40d5a67.png)

**Comment by AAnkudovich** [comment]
It wouldnt be on the level page but on members list 

What payment gateway are you using as this looks a bit different to anything i have seen

**Comment by aakankshk** [comment]
please check above comment i have upload list page and i am using stripe

**Comment by AAnkudovich** [comment]
can you show me the following page `/wp-admin/admin.php?l&page=pmpro-memberslist&s=aakankshk`

**Comment by aakankshk** [comment]
Thanks now it is showing 

![image](https://cloud.githubusercontent.com/assets/25951825/23210051/b8b92446-f8b0-11e6-8b87-1b7726d2cb4a.png)

**Comment by aakankshk** [comment]
but i am still unclear about this  why my members are level are blank list

![image](https://cloud.githubusercontent.com/assets/25951825/23210101/f497ee98-f8b0-11e6-9648-553ca6909c01.png)

if i want add two membership as group how can i add

**Comment by AAnkudovich** [comment]
The addon seems to have flaws and doesnt seem to be supported by original authors so we are unable to obtain a reason for it

**Comment by AAnkudovich** [comment]
I almost sertain in the current state of the plugin you cant

**Comment by aakankshk** [comment]
ok and even membership level page having a issue is it default behavior of this plugin

![image](https://cloud.githubusercontent.com/assets/25951825/23210361/156ffd30-f8b2-11e6-8ee5-4e5222cf3036.png)

**Comment by aakankshk** [comment]
before i can see all member level added now it is blank

**Comment by AAnkudovich** [comment]
i created a custom UI so i would not see it at all

**Comment by aakankshk** [comment]
do you have any demo i can see please?

**Comment by AAnkudovich** [comment]
/membership-account/

![screen shot 2017-02-22 at 11 58 03](https://cloud.githubusercontent.com/assets/10615682/23210585/31793ec6-f8f6-11e6-9ad1-e8e2fad51694.png)



And i overwrote /membership-account/membership-levels/ to act as a pricing table 

![screen shot 2017-02-22 at 11 57 15](https://cloud.githubusercontent.com/assets/10615682/23210572/177d258c-f8f6-11e6-8834-cb0c1be258d9.png)

**Comment by aakankshk** [comment]
ok i have already my site having 200+member and 5 membership level and if i use this addon does not it affect to my site or loose any data?

**Comment by AAnkudovich** [comment]
i am on similar numbers and lost nothing . i would advise testing it locally rather than going live with all of it right away. issue i opened a week or so ago is my only minus to this as memberships stopped to expire but the solution was given there. also backticks need to be fixed same as the open pull request on it

**Comment by aakankshk** [comment]
ok thanks for all help i looking forward to implement this as soon as possible if having issue i will add new issue

once again thank you so much for all help and support

**Comment by aakankshk** [comment]
thanks

**Comment by AAnkudovich** [comment]
Glad to be of assistance
